test/qa_settings.cpp

```diff
@@ -15,6 +15,21 @@ template<>
 auto boost::ut::cfg<boost::ut::override> = boost::ut::runner<boost::ut::reporter<>>{};
 #endif
 
+template <typename T>
```

Comment:
Do we want this somewhere more general in the project? Fmt does not want to include a default one because it is simple to define in user-code and there was a proposal to add it to std::format, which was dropped from c++23.
If yes, where would we put it?

Reply:
On the one hand, I agree that this is very common and that it is tempting to have a global definition, on the other hand, since this library will be used by many (who may have conflicting definitions) and this is only (for now) used in a unit-test context, I'd tend to not include this yet.

Also, this would IMHO expand the maintenance surface/library API. It's easy to add but unless this becomes critical to the core functionality, I'd leave it out.

We may eventually add an optional convenience library that contains all of these nick nags and that user may include in their top-level application but as a core feature this seems to be a bit heavy and also dangerous (due to potential conflict with other definitions).